test/leak/async_hooks.js

```diff
@@ -0,0 +1,109 @@
+'use strict'
+
+const tracer = require('../../')
+
+const SCOPE = require('../../src/scope/new/scope')
+
+tracer.init({
+  scope: SCOPE
```

Comment:
I think this was a later option provided to Config that doesn't have an effect as is.  The scope manager is determined by the `SIGNALFX_CONTEXT_PROPAGATION` flag being false.